lib/active_model_serializers/adapter/attributes.rb

```diff
@@ -8,7 +8,7 @@ def initialize(serializer, options = {})
       end
 
       def serializable_hash(options = nil)
-        options ||= {}
+        options = super
```

Comment:
I actually find this pattern quite weird :smile: I mean, when I read `super` I would expect it to return a `Hash`.